be/src/storage/lake/schema_change.cpp

```diff
@@ -372,8 +372,15 @@ Status SchemaChangeHandler::convert_historical_rowsets(const SchemaChangeParams&
     // copy delete vector files if necessary
     if (op_schema_change->linked_segment() && base_metadata->has_delvec_meta()) {
         for (const auto& delvec : base_metadata->delvec_meta().delvecs()) {
-            auto src = base_tablet->delvec_location(delvec.second.version());
-            auto dst = new_tablet->delvec_location(delvec.second.version());
+            auto itr_src = base_metadata->delvec_meta().version_to_delvec().find(delvec.second.version());
```

Comment:
The whole for branch can be deleted now.

Reply:
yes, you are right